### ISSUE TITLE
No matter how hard I try, I can't get a localized string.

### ISSUE THREAD

**Issue by lemkoleg** [icerockdev/moko-resources#766]
I do everything according to the instructions. Resources are generated, I use them. But, the application outputs lines in English. What could I be doing wrong? Testing on Android. Compose MPP. I also use stringResource and res.getString(LocalContext.current). Doesn't matter

**Comment by mndsl** [comment]
The localization is based on the current language of the phone.
See [Example 6 - Select localization in runtime](https://github.com/icerockdev/moko-resources?tab=readme-ov-file#example-6---select-localization-in-runtime)

**Comment by lemkoleg** [comment]
> The localization is based on the current language of the phone. See [Example 6 - Select localization in runtime](https://github.com/icerockdev/moko-resources?tab=readme-ov-file#example-6---select-localization-in-runtime)

Now I see the problem. It is related to the Ukrainian language (ua). Moko does not support all languages?

**Comment by lemkoleg** [comment]
> > The localization is based on the current language of the phone. See [Example 6 - Select localization in runtime](https://github.com/icerockdev/moko-resources?tab=readme-ov-file#example-6---select-localization-in-runtime)
> 
> Now I see the problem. It is related to the Ukrainian language (ua). Moko does not support all languages?

No, everything is fine with the language. If you install it forcibly, it works. Moko does not understand when the system language is Ukrainian.
